subprojects/packagefiles/libebml/meson.build

```diff
@@ -0,0 +1,88 @@
+project('libebml', 'cpp',
+  version : '1.4.2',
+  default_options : ['warning_level=3', 'cpp_std=c++14'])
+
+#hacks to fix compilation with GCC11. They go away with the next version
+run_command('sed', 's/cassert/limits/g', '-i', 'src/EbmlString.cpp', check : true)
+run_command('sed', 's/cassert/limits/g', '-i', 'src/EbmlUnicodeString.cpp', check : true)
```

Comment:
`sed` may be quite unavailable on Windows.

On BSD systems, and on macOS which uses the BSD sed, this will fail -- `src/EbmlString.cpp` is parsed as the value of `-i <extension>`. You need a different syntax for using GNU sed or BSD sed, and POSIX doesn't standardize any way to edit a file in place.

This also modifies the source directory which seems... very gross. Handling it as a custom_target which creates a new, modified file in builddir/ might make more sense...

And for context, this is a "backport" of https://github.com/Matroska-Org/libebml/pull/81
So... is there a status update on when upstream is going to release this?

Reply:
There is not.

My initial attempt was to use a .patch file and using patch but I guess even that is unavailable on Windows...